platform/ios/export/export_plugin.cpp

```diff
@@ -1850,7 +1874,17 @@ Error EditorExportPlatformIOS::_export_ios_plugins(const Ref<EditorExportPreset>
 			added_linked_dependenciy_names.push_back(name);
 			plugin_linked_dependencies.push_back(dependency);
 		}
+		
+		for (int j = 0; j < plugin.pods_dependencies.size(); j++) {
+			String dependency = plugin.pods_dependencies[j];
+			
+			if (plugin_pods_dependencies.has(dependency)) {
+				continue;
+			}
 
+			plugin_pods_dependencies.push_back(dependency);
+		}
+		
```

Comment:
Would love to hear of a better way to implement this. Currently I mostly re-used the code that the other dependencies used as well but it feels a bit off to me.